tests/contents/imagetracks/infos.js

```diff
@@ -8,5 +8,5 @@ const BASE_URL =
   "/imagetracks/";
 
 export default {
-  url: BASE_URL + "example.bif",
+  url: BASE_URL + "image.bif",
```

Comment:
did the file has been renamed?
I see it's named `tests/contents/imagetracks/example.bif`, I'm not sure this file will be resolved correctly.

Reply:
Yeah in the corresponding `url.js` (`tests/contents/imagetracks/urls.mjs`) it is declared as `image.bif` confusingly enough.

I thought that this way it was clearer that it's not the actual path, only the actual URL, so I did not rename the file.

Before that wasn't a problem because that URL was never used.

Reply:
Just to give a precision, we rely here on `url.js` files instead of just statically serving `tests/contents` to do many things:
  - We can configure the `Content-Type` explicitely in that `url.js` file instead of inferring it in a static server from e.g. a file's extension or its first bytes
  - We can on demand update / create files each time some `url` are requested. This is what we do with video and audio segments by just keeping one version of it and then updating the ISOBMFF's `tfdt` and `trun` boxes respectively to update the start and end times of that same segment depending on its URL (e.g. `1.mp4` will be `0-2`, `2.mp4` `2-4, `3.mp4` `3-4` etc.)